marklogic-data-hub-central/ui/e2e/cypress/integration/explore/entitiesSidepanel.spec.tsx

```diff
@@ -30,12 +30,12 @@ describe("Test '/Explore' left sidebar", () => {
     Cypress.Cookies.preserveOnce("HubCentralSession");
     cy.restoreLocalStorage().then(() => {
       cy.log(`**Go to Explore section**`);
```

Comment:
Here the then can be removed.
And the log line added to the it below, next to the vist.

Reply:
Good point, removed.